Cake.Frosting.Issues.Recipe/Cake.Frosting.Issues.Recipe/Context/State/IssuesState.cs

```diff
@@ -17,14 +17,20 @@ public class IssuesState : IIssuesState
 
     private readonly List<(IIssueProvider, string)> issueProvidersAndRuns = [];
 
+    private DirectoryPath? projectRootDirectoryOverride;
```

Comment:
We don't use nullable reference types in this project


```suggestion
    private DirectoryPath projectRootDirectoryOverride;
```